From 8b1d66a89a7214e040d2ca557e17c24404241a3a Mon Sep 17 00:00:00 2001
From: "Aaron T. Myers" <atm@apache.org>
Date: Thu, 18 Sep 2014 17:59:36 -0700
Subject: [PATCH 424/791] HADOOP-11109. Site build is broken. Contributed by
 Jian He. (cherry picked from commit
 0e2b64f2029cabbbf05a132625244427f8bf9518)

(cherry picked from commit 71e6a4a735222c25bd0be7f6811863613ece3114)
---
 .../hadoop-kms/src/site/apt/index.apt.vm           |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm b/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm
index 5ab0bbe..2e8405f 100644
--- a/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm
+++ b/hadoop-common-project/hadoop-kms/src/site/apt/index.apt.vm
@@ -626,7 +626,7 @@ $ keytool -genkey -alias tomcat -keyalg RSA
 
   <<NOTE:>> If using HTTPS, the SSL certificate used by the KMS instance must
   be configured to support multiple hostnames (see Java 7
-  <<<keytool>> SAN extension support for details on how to do this).
+  <<<keytool>>> SAN extension support for details on how to do this).
 
 *** HTTP Authentication Signature
 
-- 
1.7.9.5

